src/containers/BooksForm.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const BooksForm = () => {
+  const bookCat = ['Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi', 'Sport'];
+  return (
+    <form>
+      <input type="text" placeholder="Title" />
+      <label htmlFor="categories">
+        Select your favorite book in the following category:
+        <select id="categories">
+          <option value="biography">{bookCat[0]}</option>
+          <option value="history">{bookCat[1]}</option>
+          <option value="horror">{bookCat[2]}</option>
+          <option value="kids">{bookCat[3]}</option>
+          <option value="learning">{bookCat[4]}</option>
+          <option value="sci-fi">{bookCat[5]}</option>
+          <option value="sport">{bookCat[6]}</option>
```

Comment:
It is best practice to use the higher-order function, map, to loop through the bookCat array to implement the drop down. Go through this video also: https://www.youtube.com/watch?v=G3BS3sh3D8Q